src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
+
+    private static final Random random = new Random();
+
     public String getRandomColor() {
```

Comment:
According to the checklist, access modifiers should be specified explicitly. The method getRandomColor() should have an access modifier to define its visibility properly.